### PR TITLE
Rich Rogers author page

### PR DESCRIPTION
**Pull request by slrrogers**
Please add a direct link to your post here:

Added authors page only 

Have you (please tick each box to show completion): N/A

 - [ ] Added your blog post to a single category?
 - [ ] Added a brief summary for your post? Summaries should be roughly two sentences in length and give potential readers a good idea of the contents of your post.
 - [ ] Checked that the build passes?
 - [ ] Checked your spelling (you can use `npm install` followed by `npx mdspell "**/{FILE_NAME}.md" --en-gb -a -n -x -t` if that's your thing)
 - [ ] Ensured that your author profile contains a profile image, and a brief description of yourself? (make it more interesting than just your job title!)
 - [ ] Optimised any images in your post? They should be less than 100KBytes as a general guide.

Posts are reviewed / approved by your Regional Tech Lead.

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -784,6 +785,11 @@ authors:
     author-summary: "<p>I'm a software developer at Scott Logic, Edinburgh. I joined in August 2017 after four years studying Computer Science and Mathematics at the University of Edinburgh.</p>"
     twitter-url: "https://twitter.com/trrhodes"
     picture: picture.jpg
+  rrogers:
+    name: "Rich Rogers"
+    twitter-handle: richrtesting
+    author-summary: "<p>I am the Testing Capability Lead for Scott Logic. I have been involved in Testing and software delivery for 25 years, working for organisations in the UK and Australia.</p>"
```
Comment (jstrong-scottlogic) [inline _data/authors.yml]: Don't think 'testing' needs capitalisation in 'I have been involved in Testing'

File: _data/authors.yml [hunk 2]
```diff
@@ -784,6 +785,11 @@ authors:
     author-summary: "<p>I'm a software developer at Scott Logic, Edinburgh. I joined in August 2017 after four years studying Computer Science and Mathematics at the University of Edinburgh.</p>"
     twitter-url: "https://twitter.com/trrhodes"
     picture: picture.jpg
+  rrogers:
+    name: "Rich Rogers"
+    twitter-handle: richrtesting
```
Comment (sshiells-scottlogic) [inline _data/authors.yml]: A quick look at other entries also specifiy a twitter-url. Do you want to add that as well?

**Review by sshiells-scottlogic — COMMENTED** [review]
LGTM. 1 comment regarding the twitter-url but other than that all looks fine as far as I can tell.

File: _data/authors.yml (hunk 2 above)
Comment (slrrogers) [reply _data/authors.yml]: I may just remove the twitter handle actually.

File: _data/authors.yml (hunk 1 above)
Comment (slrrogers) [reply _data/authors.yml]: Good spot :)

**Review by sshiells-scottlogic — APPROVED** [review]

**Review by jstrong-scottlogic — APPROVED** [review]

**Pull request merged**